### PR TITLE
Simpler #3440

### PR DESCRIPTION
**Pull request by embray**
This is really all that's needed--no need worrying about differences between Python versions in handling of relative imports.  The stuff in #3440 is overkill.

### CODE REVIEW

File: astropy/_erfa/setup_package.py [hunk 1]
```diff
@@ -60,11 +42,13 @@ def preprocess_source():
         erfa_mtime = max(os.path.getmtime(filename) for filename in SRC_FILES)
         gen_mtime = min(os.path.getmtime(filename) for filename in GEN_FILES)
 
-        release = is_release()
+        version = get_pkg_version_module('astropy')
+        import pdb; pdb.set_trace()
```
Comment (embray) [inline astropy/_erfa/setup_package.py]: D'oh!~

**Comment by embray** [comment]
Ping @eteq for :+1:/:-1: as the author of the code being replaced.

**Comment by eteq** [comment]
Looks good - I tested and it does indeed behave correctly with a fake "release" for py2.7 and 3.4 .  

Only reason I didn't use `get_pkg_version_module` is that I didn't know about it.  So this is definitely better. Merging!

**Comment by embray** [comment]
I didn't know about it either, but I suspected it existed and was pleased when these suspicions were validated :grin:

**Pull request merged**